services/spar/src/Spar/Scim/User.hs

```diff
@@ -627,25 +629,63 @@ synthesizeStoredUserU usr = do
       (userDisplayName usr)
       handle
       richInfo
-      accStatus
+      (Just accStatus)
       createdAt
       lastUpdatedAt
       baseuri
   lift $ writeState tid (BT.userId usr) accessTimes (BT.userManagedBy usr) storedUser
   pure storedUser
 
+synthesizeStoredUserI :: Inv.Invitation -> Scim.ScimHandler Spar (Scim.StoredUser ST.SparTag)
```

Comment:
dont be shy of characters.    `s/I/FromInvitation`

Reply:
if you like 1958a8c, this comment doesn't apply any more.  let me know if you don't, then i'll revert it and give this the longer name.